src/ipahealthcheck/dogtag/ca.py

```diff
@@ -102,10 +102,13 @@ def check(self):
             logger.debug('CA is not configured, skipping connectivity check')
             return
 
+        cert = x509.load_certificate_from_file(paths.IPA_CA_CRT)
```

Comment:
The file `/etc/ipa/ca.crt` may contain multiple certificates, including external ones added with `ipa-cacert-manage install`, and the `load_certificate_from_file` function reads the first one if there are multiple certs in the file. As a consequence, serial_number could correspond to a cert that is not IPA CA.
Another way could be to pick any cert from ou=authorities,ou=ca,o=ipaca (the external CAs are not added there).

I am not sure it's a big deal, given that the code handles the error 'not found', but in this case do we really need to extract a serial number instead of hardcoding "1"?